sources/packages/backend/apps/queue-consumers/src/processors/virus-scan/virus-scan.processor.ts

```diff
@@ -36,7 +39,7 @@ export class VirusScanProcessor {
     } catch (error: unknown) {
       if (error instanceof CustomNamedError) {
         const errorMessage = error.message;
-        if (error.name === FILE_NOT_FOUND) {
+        if ([FILE_NOT_FOUND, EMPTY_FILE].includes(error.name)) {
```

Comment:
Very minor. please update the comment underneath.

Reply:
Makes sense, updated.